core/clients/commandclients/command-clients.go

```diff
@@ -19,15 +19,17 @@ package commandclients
 
 import (
 	"errors"
-	"fmt"
 	"io/ioutil"
 	"net/http"
 	"strings"
 
 	"github.com/edgexfoundry/edgex-go/core/clients/metadataclients"
 	"github.com/edgexfoundry/edgex-go/core/domain/models"
+	"github.com/edgexfoundry/edgex-go/support/logging-client"
```

Comment:
if we use this logging client across all projects maybe it should be under pkg dir?